Lp/src/components/AddLpModal.tsx

```
@@ -0,0 +1,161 @@
+                <button
+                  className="w-30 bg-gray-500 rounded-md cursor-pointer"
+                  onClick={handleAddTag}
+                >
+                  Add
+                </button>
+              </div>
+              <div className="flex flex-wrap gap-2">
+                {lpTagList.map((tag) => (
+                  <div className="flex gap-2 border border-white rounded p-2">
+                    {tag}
+                    <button
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)